rtl/eclrtl/eclhelper_dyn.cpp

```diff
@@ -106,14 +106,14 @@ static int compareOffsets(const unsigned *a, const unsigned *b)
 class ECLRTL_API CDynamicDiskReadArg : public CThorDiskReadArg
 {
 public:
-    CDynamicDiskReadArg(const char *_fileName, IOutputMetaData *_in, IOutputMetaData *_out, unsigned __int64 _chooseN, unsigned __int64 _skipN, unsigned __int64 _rowLimit)
-        : fileName(_fileName), in(_in), out(_out), chooseN(_chooseN), skipN(_skipN), rowLimit(_rowLimit)
+    CDynamicDiskReadArg(const char *_fileName, IOutputMetaData *_in, IOutputMetaData *_projected, IOutputMetaData *_out, unsigned __int64 _chooseN, unsigned __int64 _skipN, unsigned __int64 _rowLimit)
+        : fileName(_fileName), in(_in), projected(_projected), out(_out), chooseN(_chooseN), skipN(_skipN), rowLimit(_rowLimit)
```

Comment:
```
    virtual IOutputMetaData * queryProjectedDiskRecordSize() override final
    {
        return in;
    }
```

looks wrong now, should return 'projected' ?

Reply:
yes I will fix.